src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -95,6 +97,10 @@
 		this.minHeight = CubicChunks.Config.DEFAULT_MIN_WORLD_HEIGHT;
 		//don't want to make world implement IConfigChangeListener
 		CubicChunks.addConfigChangeListener(this);
+
+		if(!(this.provider instanceof ICubicWorldProvider)) { // if the provider is vanilla, wrap it
+			this.provider = new VanillaCubicProvider(this, provider, null);
```

Comment:
Correct me if wrong, but `null` is now being passed instead of `((ChunkProviderServer) this.chunkProvider).chunkGenerator` for `WorldServer`, no?
If so, intended?